.travis/preinstall

```diff
@@ -16,8 +16,9 @@ SCRIPTDIR=$(readlink -f $(dirname $0))
 # Add python3-selinux package (needed by Molecule on selinux enabled systems,
 # because Molecule is using `copy` and `file` Ansible modules to setup the
 # container).
+# Add gi packages so tests can run
 if lsr_venv_python_matches_system_python; then
-  LSR_EXTRA_PACKAGES='python3-selinux'
+  LSR_EXTRA_PACKAGES='python3-selinux python3-gi python-gi gir1.2-gtk-3.0 python-gi-cairo python3-gi-cairo'
```

Comment:
Are really all of these packages required? python3-gi/python-gi should probably be enough.

Reply:
Even with these packages i still can’t get gi to import. Will try some other stuff. Maybe I can just mock the calls to NM() too

Reply:
removed these now